chain/exchange/protocol.go

```diff
@@ -25,14 +33,16 @@ const BlockSyncProtocolID = "/fil/sync/blk/0.0.1"
 //   qualifier to avoid "const initializer [...] is not a constant" error.)
 var MaxRequestLength = uint64(build.ForkLengthThreshold)
 
-// Extracted constants from the code.
-// FIXME: Should be reviewed and confirmed.
-const SUCCESS_PEER_TAG_VALUE = 25
-const WRITE_REQ_DEADLINE = 5 * time.Second
-const READ_RES_DEADLINE = WRITE_REQ_DEADLINE
-const READ_RES_MIN_SPEED = 50 << 10
-const SHUFFLE_PEERS_PREFIX = 5
-const WRITE_RES_DEADLINE = 60 * time.Second
+const (
+	// Extracted constants from the code.
+	// FIXME: Should be reviewed and confirmed.
+	SuccessPeerTagValue = 25
+	WriteReqDeadline    = 5 * time.Second
+	ReadResDeadline     = WriteReqDeadline
```

Comment:
Yeah, interested to know if we actually want these to be linked or if we just did not want to see repetition 😅...

If no then we should unlink and set to `5 * time.Second` and if yes then there should be a comment to explain the linking is intentional!

Reply:
I left all this logically unchanged, but yeah, this is a bit odd. Theoretically the response read timeout should be a bit larger than the write timeout, to account for the fact that this is an RPC and the remote party has work to do before they can actually return a response.

Reply:
I probably linked them when extracting the constants, fine to decouple if you want.